Makefile

```diff
@@ -1,4 +1,4 @@
-all: pep8 pylint tests build
+all: build
```

Comment:
http://stackoverflow.com/questions/2514903/what-does-all-stand-for-in-a-makefile

tl;dr 
all is usually the default target, and it should *build* all products.